tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es2017",
```

Comment:
May I ask you why it's needed?

Reply:
I didn't need it right now. Thank you for checking!

Reply:
Thanks for updating, but the error below appeared in `src/components/page/GroupsPage.tsx`

> Top-level 'await' expressions are only allowed when the 'module' option is set to 'es2022', 'esnext', 'system', 'node16', or 'nodenext', and the 'target' option is set to 'es2017' or higher.

Reply:
Sorry to bother you. I fixed. Can you check again?